Gems/Multiplayer/Code/Source/Components/NetworkHitVolumesComponent.cpp

```diff
@@ -125,9 +125,18 @@ namespace Multiplayer
         EMotionFX::Integration::ActorComponentNotificationBus::Handler::BusConnect(GetEntityId());
         GetNetBindComponent()->AddEntitySyncRewindEventHandler(m_syncRewindHandler);
         GetNetBindComponent()->AddEntityPreRenderEventHandler(m_preRenderHandler);
-        m_physicsCharacter = Physics::CharacterRequestBus::FindFirstHandler(GetEntityId());
         GetTransformComponent()->BindTransformChangedEventHandler(m_transformChangedHandler);
         OnTransformUpdate(GetTransformComponent()->GetWorldTM());
+
+        // During activation the character controller is not created yet.
+        // Connect to CharacterNotificationBus to listen when it's activated after creation.
+        Physics::CharacterNotificationBus::Handler::BusConnect(GetEntityId());
+    }
+
+    void NetworkHitVolumesComponent::OnCharacterActivated([[maybe_unused]] const AZ::EntityId& entityId)
+    {
+        m_physicsCharacter = Physics::CharacterRequestBus::FindFirstHandler(GetEntityId());
+        Physics::CharacterNotificationBus::Handler::BusDisconnect();
     }
```

Comment:
In this case, since the character is being cached you'd want to listen to `OnCharacterDeactivated`  to reset `m_physicsCharacter` to `nullptr`. Also not disconnect from the bus here.

Reply:
Done